plugin-gradle/src/main/java/com/diffplug/gradle/spotless/KotlinExtension.java

```diff
@@ -17,28 +17,11 @@
 
 import static com.diffplug.spotless.kotlin.KotlinConstants.LICENSE_HEADER_DELIMITER;
 
-import java.io.File;
-import java.io.IOException;
-import java.util.Collections;
-import java.util.Map;
-import java.util.Objects;
-import java.util.function.Consumer;
-
-import javax.annotation.Nullable;
 import javax.inject.Inject;
 
 import org.gradle.api.tasks.SourceSet;
 
-import com.diffplug.common.collect.ImmutableSortedMap;
-import com.diffplug.spotless.FileSignature;
-import com.diffplug.spotless.FormatterStep;
-import com.diffplug.spotless.kotlin.DiktatStep;
-import com.diffplug.spotless.kotlin.KtLintStep;
-import com.diffplug.spotless.kotlin.KtfmtStep;
-import com.diffplug.spotless.kotlin.KtfmtStep.KtfmtFormattingOptions;
-import com.diffplug.spotless.kotlin.KtfmtStep.Style;
-
-public class KotlinExtension extends FormatExtension implements HasBuiltinDelimiterForLicense, JvmLang {
+public class KotlinExtension extends BaseKotlinExtension implements HasBuiltinDelimiterForLicense, JvmLang {
```

Comment:
Should we implement `HasBuiltinDelimiterForLicense` for `KotlinGradleExtension`? If so we can move this into `BaseKotlinExtension` as well.

Reply:
Same problem as for Groovy imo.